packages/rapidpro/src/Adaptor.js

```diff
@@ -2,7 +2,7 @@ import {
   execute as commonExecute,
   composeNextState,
   expandReferences,
-  http,
+  http, // Important: this is the OLD axios-based http.
```

Comment:
Rapdpro is version locked to an old version of common and should not get a new release here